Training25/Program.cs

```diff
@@ -3,11 +3,62 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program that prints sorted array by keeping the elements matching special character S
+// to the last of the array
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
 namespace Training25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      do {
+         Clear ();
+         Write ("Enter array of letters: ");
+         var (str, isValid) = GetArray ();
+         if (isValid) {
+            Write ("Enter special character: ");
+            var splChar = GetSplChar ();
+            WriteLine ("\nPress 'D' to sort in descending order");
+            WriteLine (SortAndSwap (str, splChar, ReadKey (true).Key == ConsoleKey.D));
+         } else WriteLine (str);
+         Write ("Press 'Y' to continue");
+      } while (ReadKey (true).Key == ConsoleKey.Y);
+   }
+
+   // Returns the special character to be added at the end of the string.
+   static char GetSplChar () {
+      var splChar = ReadKey ().KeyChar;
+      while (!char.IsLetter (splChar)) {
+         WriteLine ("\nPlease enter a valid character (A-Z).Enter special character: ");
+         splChar = ReadKey ().KeyChar;
+      }
+      return splChar;
+   }
+
+   // Returns the error message (or) extracted string from comma separated input.
+   static (string str, bool isValid) GetArray () {
+      var array = ReadLine ();
+      if (string.IsNullOrEmpty (array)) return ("Input cannot be empty!", false);
+      string str = "";
+      for (int i = 0, len = array.Length; i < len; i++) {

```

Comment:
```suggestion
      for (int i = 0; i < array.Length; i++) {
```
This is standard format. As 'len' is not used inside for loop declaring it is unnecessary.